### ISSUE TITLE
[FireFox] Can't upload any files

### ISSUE THREAD

**Issue by beefchimi** [ublue-os/bazzite#437]
I notice that whenever I attempt to upload a file in `FireFox` - such as attaching an image in `Gmail`, or uploading a screenshot to `GitHub` - it just doesn't work. I assume this is some `Flatpack` thing? Anyone have advice for how to get around this?

**Comment by rothgar** [comment]
This looks like it's related to https://bugzilla.mozilla.org/show_bug.cgi?id=1636774

If you open flatseal app and then search for firefox you can enable "All user files" which should let you drag and drop files into Firefox.
![image](https://github.com/ublue-os/bazzite/assets/371796/1d5feea6-1337-42e1-b64d-5c1ec31fec4c)

I just tested with gmail and this worked for me.

**Comment by beefchimi** [comment]
Yeehaw! That worked :partying_face: I haven't tried everything, but so far it looks like I can finally upload files :smile: 

Heads up for anyone doing this in the future: Enabling the `All user files` seems to have blown away any saved state from FireFox. So, the next time I launched FireFox, all my open tabs were gone and I needed to re-sign into everything. Not a big deal for me, but might be relevant for others.
